### ISSUE TITLE
Docker run problem

### ISSUE THREAD

**Issue by dotX12** [capyfile/capyfile#7]
Hello, i have some problem.

```sh
root@server:/home/boot/capi/capyfile# docker run \
--name capyfile_server \
--mount type=bind,source=/home/boot/capi/capyfile/docs.pipeline.json,target=/etc/capyfile/docs.pipeline.json \
--env CAPYFILE_SERVICE_DEFINITION_FILE=/etc/capyfile/docs.pipeline.json \
--env AWS_DOCUMENTS_BUCKET=documents \
--env aws_access_key_id \
--env aws_secret_access_key \
-p 8024:80 \
capyfile/capysvr:latest
```

```sh
exec /app/capysvr: no such file or directory
```

Why might this happen?

**Comment by sky003** [comment]
It looks like we have a missing dependency on the Alpine docker image.

If you want some fix right now, you can run this Dockerfile:
```
FROM capyfile/capysvr:1.2.2
RUN apk add --no-cache libc6-compat 
```

I'm probably going to push the fixed version later today.

**Comment by sky003** [comment]
This issue has been fixed on v1.2.3
